src/implementation/compression_test.cpp

```diff
@@ -0,0 +1,58 @@
+#include "gtest/gtest.h"
+#include "compression.cpp"
+
+vector<ll> populateVector(int n) {
+    vector<ll> v;
+    rep(i,0,n) {
+        v.push_back(rand() - RAND_MAX/2);
```

Comment:
`rand` always returns int, so `vector<ll>` is unnecessary.

Reply:
Yes, but `compress` expects a `vector<ll>`

Reply:
My fault, didn't see that :innocent: